src/DBforsimulation.jl

```diff
@@ -1,9 +1,46 @@
-export get_info_from_DB
+export fix_fknee, get_info_from_DB
 import Random
 
 
 """
-    function get_info_from_DB(db, horns, stokes)
+    function fix_fknee(f_samp_hz, tcmb_k, tatm_k, ttel_k, tnoise_k, β_hz, slope, fknee_bicocca)
```

Comment:
Instead of `fknee_bicocca`, better use something like `fknee_ref_hz`, where `ref` stands for «reference».

Reply:
Moreover, the number of parameters is really too long! Since you are interested in the overall tsys, it would be better to squash all the temperatures in one parameter, named `tsys_k`.